frontend/src/components/Menubar/Menubar.tsx

```diff
@@ -136,7 +136,7 @@ const Menubar = () => {
                 : (
                 <Name onClick={handleEditProjectName} title="Edit title">{projectName ?? 'Untitled Project'}</Name>
                   )}
-              <SaveStatus $show={isSaving}>Saving...</SaveStatus>
+              <SaveStatus $show={isSaving && !!user}>Saving...</SaveStatus>
```

Comment:
The `!!` is so its treated has a boolean.
It pleases the linter

Reply:
I was just wondering why save status is not shown for not logged in users - is it because there's no point showing save status when someone isn't logged in because it happens so quickly it would be missed? Just wanted to double check I understand this correctly

Reply:
Yeah thats basically the reason, except the animation actually runs for a while which was a bit annoying imo (think this is what #259 is talking about)